packages/apps-config/src/api/spec/realis.ts

```diff
@@ -32,7 +32,7 @@ const definitions: OverrideBundleDefinition = {
             'Diamond'
           ]
         },
-        String: 'Vec<u8>',
+        Text: 'Vec<u8>',

```

Comment:
```suggestion
        String: 'Text',
```

The JS equivalent is called Text :)

Reply:
Okey, we change

Reply:
We cant re name this type, because UI throw waning about Metadata V14